daffodil-core/src/main/scala/org/apache/daffodil/runtime1/processors/parsers/Parser.scala

```diff
@@ -227,26 +227,38 @@ abstract class CombinatorParser(override val context: RuntimeData)
   extends Parser
   with CombinatorProcessor
 
-final class SeqCompParser(context: RuntimeData, val childParsers: Array[Parser])
-  extends CombinatorParser(context) {
-  override def runtimeDependencies = Vector()
+final class SeqCompParser(
+  context: RuntimeData,
+  val childParsers: Array[Parser],
+  testAssert: Array[Parser],
+) extends CombinatorParser(context) {
+  override lazy val runtimeDependencies = Vector()
```

Comment:
All runtimeDependencies are currently defs. Is there a reason this was changed to lazy val?